.github/ISSUE_TEMPLATE/onboarding_argocd.yaml

```diff
@@ -0,0 +1,101 @@
+name: "Onboard to Continuous Deployment"
+description: "Request a new ArgoCD access, project, application, or add a new
+user to an existing ArgoCD Project."
+title: "PROJECT: <name>"
+labels: [kind/onboarding, area/argocd]
+assignees: [first-operator]
+body:
+    - type: checkboxes
+      id: argocd-options
+      attributes:
+          label: Select all that apply
+          options:
+          - label: Create a new ArgoCD project for existing team
+          - label: Enlist an application to be deployed automatically via ArgoCD
+          - label: Add a new user to an ArgoCD Project
+    - type: markdown
+      attributes:
+          value: "# Creating a new project"
+    - type: input
+      id: team-name
+      attributes:
+          label: Team name
+          description: Please tell us your team name, which corresponds to [an already available user `Group`](https://github.com/operate-first/apps/tree/master/cluster-scope/base/groups).
+          placeholder: team-name
+    - type: dropdown
+      id: target-cluster
+      attributes:
+          label: Target cluster
+          description: Choose an environment from [this list](https://github.com/operate-first/apps/tree/master/cluster-scope/overlays)
```

Comment:
can you get rid of link from "this list" since it's now being provided within the drop down.